src/PHPStan/Reflection/TypeToCallReflectionResolver/TypeToCallReflectionResolverRegistry.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Core\PHPStan\Reflection\TypeToCallReflectionResolver;
+
+use PHPStan\Reflection\ClassMemberAccessAnswerer;
+use PHPStan\Reflection\FunctionReflection;
+use PHPStan\Reflection\MethodReflection;
+use PHPStan\Type\Type;
+
+final class TypeToCallReflectionResolverRegistry
+{
+    /**
+     * @var iterable|TypeToCallReflectionResolverInterface[]
+     */
+    private $resolvers;
```

Comment:
```suggestion
    private $resolvers = [];
```

To 100 % prevent nullable foreaching. I saw it breaking code in constructor of *forgotten* init in ctors

Reply:
Done.